CppCoreGuidelines.md

```diff
@@ -3528,13 +3528,31 @@ There is not a choice when a set of functions are used to do a semantically equi
 
     ???
 
-### <a name="Rf-reference-capture"></a>F.52: Prefer capturing by reference in lambdas that will be used locally, including passed to algorithms
+### <a name="Rf-reference-capture"></a>F.52: When a lambda stays in local scope, prefer capturing by reference, including passed to algorithms
 
 ##### Reason
 
-For efficiency and correctness, you nearly always want to capture by reference when using the lambda locally. This includes when writing or calling parallel algorithms that are local because they join before returning.
+When a lambda is only used locally, capturing by reference is always more efficient.  This includes calling parallel algorithms because they join before returning.
```

Comment:
Yes, I did.  I dropped it intentionally because I don't see how correctness can be influenced in a situation where value semantics are the rule.

I may be mistaken, though.  I'll post a question on StackOverflow.